projects/fourier/lib/fourier/commands/update.rb

```diff
@@ -18,10 +49,40 @@ def swiftlint
       end
 
       desc "xcbeautify", "Update the vendored xcbeautify binary"
+      option(
+        :xcode_version,
+        desc: %(
+          The version of Xcode to use to build tuist.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
```

Comment:
As Danielle has mentioned, let's use the version in the `.xcode-version` file

Reply:
Sorry I don't quite understand. Should I remove these parameters? How do you want me to use the `.xcode-version` file?

Reply:
Yep 👍